src/main/java/com/qdesrame/openapi/diff/model/ChangedExtensibleEnum.java

```diff
@@ -0,0 +1,18 @@
+package com.qdesrame.openapi.diff.model;
+
+import java.util.List;
+
+public class ChangedExtensibleEnum<T> extends ChangedList<T> {
+
+  public ChangedExtensibleEnum(List<T> oldValue, List<T> newValue, DiffContext context) {
+    super(oldValue, newValue, context);
+  }
+
+  @Override
+  public DiffResult isItemsChanged() {
+    if (this.getMissing().isEmpty()) {
```

Comment:
Only incompatible if an enum value was removed.

Reply:
could just add that as a comment :)

Reply:
Good deal